### PR TITLE
Globals: Added variableCasing and separatorSymbol setting

### PR DESCRIPTION
**Pull request by thim81**
Linked to #540

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -627,7 +627,9 @@ The configuration defined in the `globals` will be executed on the full Postman
 
 #### globals options
 
-- **stripResponseExamples (Boolean | optional)** : Strip the response examples from the generated Postman collection.
+- **stripResponseExamples (Default: false)** : Strip the response examples from the generated Postman collection.
+- **variableCasing** : Change the casing of the auto-generated Postman variables. Supported values are: `camelCase`, `pascalCase`, `kebabCase`, `trainCase`, `snakeCase`, `adaCase`, `constantCase`, `cobolCase`, `dotNotation`, `lowerCase`, `upperCase`
+- **separatorSymbol (Default: "::")** : Change the separator symbol for the auto-generated Postman testName description (Example: `[GET]::/crm/leads - Status code is 2xx`). Helpfull when using the [postman-to-k6](https://github.com/apideck-libraries/postman-to-k6) converter.
```
Comment (nicklloyd) [inline README.md:632]: Helpfull -> Helpful

Comment (thim81) [reply README.md]: good catch.

File: src/application/tests/testResponseHeaderContent.ts [hunk 2]
```diff
@@ -16,14 +17,17 @@ export const testResponseHeaderContent = (
     let pmTestMaxLength = ''
     let pmTestAssert = ''
 
+    // Separator
+    const split = config?.separatorSymbol || '::'
```
Comment (nicklloyd) [inline src/application/tests/testResponseHeaderContent.ts:21]: nullish coalescence

File: src/application/tests/testResponseJsonSchema.ts [hunk 3]
```diff
@@ -24,6 +25,8 @@ export const testResponseJsonSchema = (
   // deletes nullable and adds "null" to type array if nullable is true
   jsonSchema = convertUnsupportedJsonSchemaProperties(jsonSchema)
 
+  const split = config?.separatorSymbol || '::'
```
Comment (nicklloyd) [inline src/application/tests/testResponseJsonSchema.ts:28]: nullish coalescence

File: src/application/tests/testResponseStatusSuccess.ts [hunk 4]
```diff
@@ -1,13 +1,18 @@
...
 import { PostmanMappedOperation } from '../../postman'
+import { GlobalConfig } from 'types'
 
 export const testResponseStatusSuccess = (
-  pmOperation: PostmanMappedOperation
+  pmOperation: PostmanMappedOperation,
+  config?: GlobalConfig
 ): PostmanMappedOperation => {
+  // Separator
+  const split = config?.separatorSymbol || '::'
```
Comment (nicklloyd) [inline src/application/tests/testResponseStatusSuccess.ts:10]: nullish coalescence

File: src/application/tests/testResponseTime.ts [hunk 5]
```diff
@@ -1,21 +1,25 @@
...
-  pmOperation: PostmanMappedOperation
+  pmOperation: PostmanMappedOperation,
+  config?: GlobalConfig
   // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
 ): PostmanMappedOperation => {
   // Early exit if no maxMs defined
   if (!responseTime?.maxMs) return pmOperation
 
+  // Separator
+  const split = config?.separatorSymbol || '::'
```
Comment (nicklloyd) [inline src/application/tests/testResponseTime.ts:15]: nullish coalescence

File: src/application/variables/assignVarFromResponseHeader.ts [hunk 6]
```diff
@@ -37,7 +40,11 @@ export const assignVarFromResponseHeader = (
   // Set variable name
   const opsRef = pmOperation.id ? pmOperation.id : pmOperation.pathVar
   const varProp = varSetting.responseHeaderProp
-  const varName = varSetting.name ? varSetting.name : opsRef + '.' + varProp
+  const defaultVarName = `${opsRef}.${varProp}`
+  const casedVarName = settings?.variableCasing
+    ? changeCase(defaultVarName, settings.variableCasing)
+    : defaultVarName
+  const varName = varSetting.name || casedVarName
```
Comment (nicklloyd) [inline src/application/variables/assignVarFromResponseHeader.ts:47]: nullish coalescence

**Review by nicklloyd — APPROVED** [review]

**Pull request merged**